src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/Extensions/INamedTypeSymbolExtensions.cs

```diff
@@ -417,6 +418,8 @@ static bool IsInaccessibleImplementableAccessor(IMethodSymbol? accessor, ISymbol
             Compilation compilation,
             ISymbol within)
         {
+            using var _ = PooledHashSet<string>.GetInstance(out var seenNames);
```

Comment:
❔ How often is this called? I'd like to review the relevant perf scenarios tomorrow to ensure we aren't causing a problem for many-core machines.

Reply:
it's basically called when bringing up the sig help list for an attribute :)

Reply:
If it's only called on demand for the _active_ signature help item then it's fine.